src/client/driver/command-executors/client-functions/selector-executor/index.js

```diff
@@ -60,17 +60,39 @@ function hasText (node, textRe) {
         return textRe.test(text);
     }
 
-    // Document and DocumentFragment
-    if (node.nodeType === 9 || node.nodeType === 11)
-        return !!filterNodeCollectionByText(node.childNodes, textRe).length;
+    // Document
+    if (node.nodeType === 9) {
+        // NOTE: latest version of Edge doesn't have `innerText` for `document`,
+        // `html` and `body`. So we check their children instead.
```

Comment:
I thing we should remove  `document` form comment, because no one browser have `innerText` for `document`, only for `documentElement` === `html` element